sdk/typescript/test/e2e/signature-mpc.test.ts

```diff
@@ -120,8 +122,10 @@ describe('Test key share transfer', () => {
 
 	it('should encrypt and transfer a dwallet to a newly generated public key', async () => {
 		const [encryptionKey, decryptionKey] = generate_keypair();
+		let signed_encryption_key = await toolbox.keypair.sign(new Uint8Array(encryptionKey));
```

Comment:
Please move user-logic from tests into an exported TS function that registers the key.

Reply:
Actually that function should also include the decryption key generation from keypair. If there's an open issue for this in the future, I'll approve this PR, as you choose

Reply:
I moved the user-logic from the tests into the exported TS function that registers the key, but I'm not sure wdym by the second comment